### DISCUSSION TITLE
Affiliate / referral program

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#120] [Anything Else]
"Plausible referrals for getting more trial period time. Idk but might help spread plausible more for ya and it would be if paid users signed up for longer period vs shorter for only a trial signup. I'm still paying but may be useful for others"

([imported from the old roadmap](https://plausible.nolt.io/68))

"I recommend Plausible to everyone - I’d love to capture that and use it even more in my marketing and blogging through an affiliate feature in Plausible. You send someone a link, they sign up for a plan using that link and you get a percentage as the referrer."

([imported from the old roadmap](https://plausible.nolt.io/87))

**Comment by wolfganghofer** [comment]
Affiliate would be great! For example to get a percentage or to get your own subscription plan for free when you bring new customers. What do you think, when will this be implemented? I think I could bring new customers.

**Reply by metmarkosaric** [reply]
thanks @wolfganghofer! there's no current plan to introduce an affiliate program but we are keeping the option open. something that we will need to consider is a way to introduce affiliates without needing to raise our prices plus managing it all can be time consuming especially as we'd want to make sure that everyone is upfront and discloses the relationship when sharing affiliate links. we'll keep you posted.

**Reply by dkoprowski** [reply]
Hi! I'm interested in the affiliate program as well. Leaving a comment for future updates :)

**Reply by renoseoexpert** [reply]
Just signed up for Plausable Analytics. Hopeful to see how much easier and accurate it is compared to G4. Would like to see an affilaite program so I can promote.

**Reply by renoseoexpert** [reply]
Also, I sent one unique view, and it counted it as three. Can you explain how PA counts a pageview?

**Reply by renoseoexpert** [reply]
I installed on this website [www.RenoWebDesigner.com](https://renowebdesigner.com) using Divi Wordpress theme.

**Reply by metmarkosaric** [reply]
hi @renoseoexpert! I just checked your site and it looks to track all fine for me. btw for any type of support questions, it's best to contact [our customer support directly](https://plausible.io/contact). thanks!

**Comment by calicojack** [comment]
Same here, I run https://downforeveryoneorjustme.com/ and I'd love to promote you guys on that site as we add some more features to it.

**Comment by pottz91** [comment]
Same here, would love to promote this. Maybe on a Plattform like PartnerStack?

**Comment by landwire** [comment]
Is this planned one day?

**Reply by coppinger** [reply]
> We don’t pay anyone to recommend Plausible Analytics to their audience be it via referral programs, affiliate programs or other types of sponsorships. There is nothing inherently wrong with affiliate marketing as long as it is always prominently and clearly disclosed to the people, we just don’t do it.

From their about page: https://plausible.io/about

**Comment by LushawnDev** [comment]
I'd second this! I'm a big advocate for Plausible and often recommend it, so would be interesting to see if it grows the Plausible user base by having this incentive
